tools/report.sh

```diff
@@ -104,11 +104,12 @@ fi
 
 get_masked_secrets() {
 	mkdir -p $direct/reports/"$1"
+	echo "$1"
```

Comment:
debug?

Reply:
nope, that will print the `secrets` "header" or "label" like we have for all the `resources_to_fetch`. Without it, that only prints the secrets names, so it looks like this:

```
secrets
   secret/my-cluster-clients-ca
   secret/my-cluster-clients-ca-cert
   secret/my-cluster-cluster-ca
   secret/my-cluster-cluster-ca-cert
   secret/my-cluster-cluster-operator-certs
   secret/my-cluster-entity-operator-certs
   secret/my-cluster-kafka-brokers
   secret/my-cluster-zookeeper-nodes
```
without it it's just:
```
   secret/my-cluster-clients-ca
   secret/my-cluster-clients-ca-cert
   secret/my-cluster-cluster-ca
   secret/my-cluster-cluster-ca-cert
   secret/my-cluster-cluster-operator-certs
   secret/my-cluster-entity-operator-certs
   secret/my-cluster-kafka-brokers
   secret/my-cluster-zookeeper-nodes
```